client/player.lua

```diff
@@ -0,0 +1,52 @@
+AddStateBagChangeHandler('hunger', ('player:%s'):format(cache.serverId), function(_, _, value)
+    if value == PlayerState.hunger then return end
+    SendNUIMessage({
+        update = true,
+        data = {
+            {
+                type = 'progress',
+                name = 'hunger',
+                value = value,
+                option = {
+                    stroke = value < 30 and '#881111ff' or false,

```

Comment:
The `or false` is not necessary as it turns false without it already if the conditions are not met